packages/mc-scripts/src/commands/build-vite.ts

```diff
@@ -77,6 +86,10 @@ async function run() {
       pluginSvgr(),
       pluginDynamicBaseAssetsGlobals(),
       pluginI18nMessageCompilation(),
+      process.env.ANALYZE_BUNDLE === 'true' &&
```

Comment:
This plugin create the bundles visualization size:
![Uploading Screenshot 2024-10-08 at 18.31.02.png…]()